db/controllers/articlesController.js

```diff
@@ -12,4 +12,31 @@ exports.getArticles = (req, res, next) => {
      
             next(err)
         })
+}
+
+exports.patchVote = (req, res, next) => {
+    const incPropInArray = Object.keys(req.body);
+    const incProp = incPropInArray[0];
+    const incValInArray = Object.values(req.body);
+    const incVal = incValInArray[0];
+    const articleId = Object.values(req.params); 
```

Comment:
you can get anything you need on req.body and req.params

Reply:
Thanks for the feedback. Do I need to check the name of property?